<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="map1"  default="$(find tiago_sim_integration)/maps/012/mmap.yaml"/>
    <arg name="map2"  default="$(find tiago_sim_integration)/maps/012/map_real.yaml"/>

    <node name="map_server_1" pkg="map_server" type="map_server" args="$(arg map1)" output="screen" >
        <param name="frame_id" type="str" value="map1" />
        <remap from="map" to="map1"/>
    </node>
    <node name="map_server_2" pkg="map_server" type="map_server" args="$(arg map2)" output="screen" >
        <param name="frame_id" type="str" value="map2" />
        <remap from="map" to="map2"/>
    </node>
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find tiago_sim_integration)/config/map_adjustment_tool.rviz" />
    <node type="map_adjustment_tool.py" name="map_adjustment_tool" pkg="tiago_sim_integration" output="screen" />
</launch>
